portal-impl/src/com/liferay/portal/service/impl/WorkflowDefinitionLinkLocalServiceImpl.java

```diff
@@ -382,6 +408,31 @@ public WorkflowDefinitionLink updateWorkflowDefinitionLink(
 		return workflowDefinitionLinkPersistence.update(workflowDefinitionLink);
 	}
 
+	@Override
+	public WorkflowDefinitionLink updateWorkflowDefinitionLink(
```

Comment:
It seems that your new method should work similarly to the existing one, the only difference is the get by ERC and groupId. What do you think? Try to reuse code.